src/nn/Model.js

```diff
@@ -71,7 +71,7 @@ export default class Model {
       value: null
     }
     this._operands.push(operand);
-    return ResultCode.NO_ERROR;
+    //return ResultCode.NO_ERROR;
```

Comment:
Inline comments should be separated by space from the statement. They should start with // and a single space.

https://google.github.io/styleguide/jsguide.html#formatting-comments

It's better to follow coding style.

Reply:
Done! Thanks for the reference link!